values.yaml

```diff
@@ -47,9 +47,6 @@ global:
   ## doc/charts/globals.md#configure-appconfig-settings
   ## Rails based portions of this chart share many settings
   appConfig:
-    ## doc/charts/globals.md#general-application-settings
-    enableUsagePing: false
```

Comment:
should we make this one configurable?

Reply:
Gitlab Chart enables it by default. This was an opinionated setting inherited by the google original module.

I'll set a var enabled as default

Reply:
yes it can be a nice improvement

Reply:
Done